tests/conftest.py

```diff
@@ -325,7 +333,7 @@ def zulip_emoji():
     "sender_full_name": "Foo Foo",
     "timestamp": 1520918737,
     "client": "website",
-    "recipient_id": 5780,  # FIXME Unsure
+    "recipient_id": 6076,  # FIXME Unsure
```

Comment:
There are a few other places where we have this field defined, does it make sense to remove this and the `short_name` field in another PR?
We already have typing for `message`, so we could comment about `recipient_id` there(similar to the mobile code), and for `short_name` we could simply erase that since the server has already removed it?

Reply:
This was resolved with the recently merged #1053